bower.json

```diff
@@ -14,8 +14,8 @@
     "fxa-password-strength-checker": "https://github.com/mozilla/fxa-password-strength-checker.git#d73b3ade374607e2749ee375301b0a168008b16f",
     "html5shiv": "3.7.2",
     "JavaScript-MD5": "1.1.0",
-    "jquery": "1.11.1",
-    "jquery-modal": "https://github.com/kylefox/jquery-modal.git#5e9bdc3e4de703089fba1e38ddc0f62d7de9eff7",
+    "jquery": "3.1.0",
+    "jquery-modal": "https://github.com/shane-tomlinson/jquery-modal.git#0576775d1b4590314b114386019f4c7421c77503",
```

Comment:
The version from my account allows jQuery 3.x. I have opened a patch [against the original](https://github.com/kylefox/jquery-modal/pull/177) with this change.